frontend/js/app.js

```
@@ -400,27 +416,32 @@ function initializeSSE() {
+                showToast(data.status || 'Operation complete!', 'success');
+                return;
+            }
+
+            // Handle new items notification
             if (data.new_items > 0) {
                 showToast(`Updates: ${data.new_items} new items`, 'info');
+                allTabs = await api.getTabs();
+                renderTabs(allTabs, activeTabId, { onSwitchTab: switchTab });
+
+                const affectedIds = (data.affected_tab_ids || []).map(id => parseInt(id, 10));
+                affectedIds.forEach(id => loadedTabs.delete(id));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)